uncontrast_studies/processors/distribution_of_effects_across_parameters.py

```diff
@@ -56,8 +64,10 @@ def process_sample_size_excluded(self):
         return self._aggregate_query_by_breakdown(subquery)
 
     def _aggregate_query_by_breakdown(self, filtered_subquery):
+        BIN_SIZE = 10
+        BIN_NUMBER = 10
```

Comment:
where are you using these variables?

Reply:
currently not as I've changed it to be from the front as param, I'll remove them